### ISSUE TITLE
Back link should preserve the previous selected option when used

### ISSUE THREAD

**Issue by davidc-gds** [alphagov/govuk-design-system#2802]
This issue is from May 2023 external accessibility audit report.

**Usability issue**

**Issue ID:**
DAC_Usability_Question_Pages_01

**URLs:**
https://signin.account.gov.uk/get-security-codes

## Screen shots
<img width="765" alt="Screenshot of the 'Choose how to get security codes' page from the One Login journey, showing both radio options unselected" src="https://github.com/alphagov/govuk-design-system/assets/98318778/136fcf31-a180-47fc-9f2b-c339472ae0c9">

After a user goes back to the ‘Choose how to get security codes’ page, the page is not in the state that they last saw it. The users selected radio button has been unselected, meaning the user must answer the question again in order to continue in the service.

## Current code ref(s)
`#mfaOptions`

```
<input class="govuk-radios__input" id="mfaOptions" name="mfaOptions" type="radio" value="SMS">
```

## User comments
No user comments submitted in the report

## Solution
The GOV.UK Design System states that “Make sure it takes users to the previous page they were on; in the state they last saw it”.

When a user goes back to the previous page, their chosen option should be still selected.
Redundant entry mentions that; information previously entered by or provided to the user that is required to be entered again in the same process is either:
- auto-populated, or
- available for the user to select.

Except when:
- re-entering the information is essential,
- the information is required to ensure the security of the content, or
- previously entered information is no longer valid.

It is currently unclear if this relates to a user navigating back pages to previously completed forms.
As WCAG 2.2 is still yet to be finalised it is currently unclear if this issue would fail this criterion.

**Comment by davidc-gds** [comment]
Hey @CharlotteDowns I'm thinking there's not much to do on this issue, if we've contacted the One Login team.

We already have guidance that covers this, under the ['How it works' section of the 'back link' page](https://design-system.service.gov.uk/components/back-link/#how-it-works). It seems decently well-placed in the guidance as well, in my opinion.
CC @claireashworth for a second content opinion perhaps?

Would it be reasonable to close this without further action?

**Comment by davidc-gds** [comment]
We did have a live service implement the back link without the correct back-linking, which is how we got this audit result. However, it's not clear if that service team ever read our guidance on back links, or just used the code

**Comment by davidc-gds** [comment]
The team who ran this service has solved the issue here: https://github.com/alphagov/di-authentication-frontend/pull/1075

Closing this ticket as completed!
